src/components/implementation/tests/sqlite_tests/sqlite_tests.c

```diff
@@ -0,0 +1,44 @@
+#include <cos_component.h>
+#include <llprint.h>
+#include <sqlite3.h>
+#include <cos_defkernel_api.h>
+
+void
+cos_init(void)
+{
+}
+
+int result_print(void *v, int argc, char **argv, 
+                    char **col_name) {
+    for (int i = 0; i < argc; i++) {
+        printc("%s = %s\n", col_name[i], argv[i] ? argv[i] : "NULL");
+    }
+    
+    return 0;
+}
+
+int
+main(void)
+{
+	printc("Calling sqlite functions\n");
+
+    sqlite3 *db;
+    char *err_msg = 0;
+    char *sql = "CREATE TABLE TestTable(Id INTEGER PRIMARY KEY, Value TEXT);"
+    "INSERT INTO TestTable(Value) VALUES ('TestItem1');"
```

Comment:
Personal opinion (ignore if you'd like): all strings would look better if the starting "s were aligned.